source/UIAHandler/__init__.py

```diff
@@ -1390,5 +1440,23 @@ def terminate():
 		handler.terminate()
 		handler = None
 
+
 def _isDebug():
 	return config.conf["debugLog"]["UIA"]
+
+
+def __getattr__(attrName: str) -> Any:
+	"""Module level `__getattr__` used to preserve backward compatibility."""
+	if attrName == "HorizontalTextAlignment_Left" and NVDAState._allowDeprecatedAPI():

```

Comment:
I would personally do the if check for deprecated API first and then use a match statement for the constants.

Reply:
Agreed - this makes the code more readable.